pyccel/ast/core.py

```diff
@@ -2217,39 +2217,15 @@ def __new__(
         cls,
         dtype,
         name,
-        rank=0,
-        allocatable=False,
-        is_stack_array = False,
-        is_pointer=False,
-        is_target=False,
-        is_polymorphic=None,
-        is_optional=None,
-        shape=None,
-        cls_base=None,
-        cls_parameters=None,
-        order='C',
-        precision=0,
-        is_argument=False
+        **kwargs
```

Comment:
This is not what I meant... You can add a `*` argument between `name` and `rank`.

Reply:
See https://docs.python.org/3/reference/compound_stmts.html#function-definitions